components/WeeklyPickButton/WeeklyPickButton.tsx

```diff
@@ -19,6 +19,7 @@ const WeeklyPickButton: React.FC<WeeklyPickButtonProps> = ({ team, src }) => {
           height={48}
           priority
           data-testid="team-image"
+          className="h-12 w-12"
```

Comment:
Is this responsive, though? Seems like this is always keeping it at 48px.

Reply:
That thought also crossed my mind, but when I looked at the Figma design for mobile, tablet, large desktop etc., it seems like it's still set at a fixed width and height of 48px.